### PR TITLE
Fixed more test that were flickering on the Travis CI build

### PR DESCRIPTION
**Pull request by merlimat**
### Motivation

Fixed more test that were flickering on the Travis CI build.

### Modifications

 * Disabled code coverage on every build
 * Changed timeout test to serialize execution into main thread

### Result

Travis build has been consistently passing

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by yahoocla** [comment]
CLA is valid!

File: pulsar-broker/src/test/java/com/yahoo/pulsar/broker/admin/NamespacesTest.java [hunk 1]
```diff
@@ -491,16 +491,6 @@ public void testGetBundles() throws Exception {
                 "test-bundled-namespace-1");
 
         assertEquals(responseData, bundle);
-
-        mockZookKeeper.failNow(Code.SESSIONEXPIRED);
```
Comment (rdhabalia) [inline pulsar-broker/src/test/java/com/yahoo/pulsar/broker/admin/NamespacesTest.java]: should we keep this usecase as part of test.

**Comment by rdhabalia** [comment]
looks good 👍

File: pulsar-broker/src/test/java/com/yahoo/pulsar/broker/admin/NamespacesTest.java (hunk 1 above)
Comment (merlimat) [inline pulsar-broker/src/test/java/com/yahoo/pulsar/broker/admin/NamespacesTest.java]: that portion is a bit unreliable and not that relevant anyway

**Pull request merged**